node/src/main/resources/files/limbo/server.properties

```diff
@@ -0,0 +1,30 @@
+#For explaination of what each of the options does, please visit:
+#https://github.com/LOOHP/Limbo/blob/master/src/main/resources/server.properties
+#Thu May 23 16:16:17 CEST 2024
+allow-chat=true
+allow-flight=false
+bungee-guard=false
+bungeecord=false
```

Comment:
Usually we enable bungeecord support by default. I think we should do that here too.